client/src/Components/Pages/LandingPage/HeroSection/index.js

```diff
@@ -18,9 +24,10 @@ const HeroComp = (props)=> {
                     <li>2. Privately held startup company valued at over $1 billion</li>
                 </ul>
             </div>
+            <MainButton  value="Join us" onClick = {this.onClickHandler}/>
```

Comment:
I see that you changed HeroComp from a class to a functional component - does using `this`, as in `this.onClickHandler`, still work?  I would have thought that HeroComp would need to be a class for `this.` to work.